play-json/src/main/scala/com/lightbend/lagom/scaladsl/playjson/PlayJsonSerializer.scala

```diff
@@ -159,7 +159,7 @@ private[lagom] final class PlayJsonSerializer(val system: ExtendedActorSystem, r
 }
 
 // This code is copied from JacksonJsonSerializer
-private[lagom] final object Compression {
+private[lagom] object Compression {
```

Comment:
Why do we need to remove this `final`?

Reply:
The final modifier is not required to be specified explicitly for top-level objects in Scala.

Reply:
`object`! (pours 1 extra litre of coffee) 👍